website/public/js/controllers/rootCtrl.js

```diff
@@ -215,7 +215,18 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
 
       var gems = User.user.balance * 4;
 
-      var string = (type == 'weapon') ? window.env.t('weapon') : (type == 'armor') ? window.env.t('armor') : (type == 'head') ? window.env.t('headgear') : (type == 'shield') ? window.env.t('offhand') : (type == 'back') ? window.env.t('back') : (type == 'body') ? window.env.t('body') : (type == 'headAccessory') ? window.env.t('headAccessory') : (type == 'eyewear') ? window.env.t('eyewear') : (type == 'hatchingPotions') ? window.env.t('hatchingPotion') : (type == 'eggs') ? window.env.t('eggSingular') : (type == 'quests') ? window.env.t('quest') : (item.key == 'Saddle') ? window.env.t('foodSaddleText').toLowerCase() : type; // FIXME this is ugly but temporary, once the purchase modal is done this will be removed
```

Comment:
I really don't like the idea of having what's basically a duplicate translatable string; it's just extra work for all the translators. I don't think having it in lower case is that important - the lower case version is visible in only the popup when you buy the saddle, and the upper case version is used everywhere else. Let's just keep using the existing "Saddle" string and not bother about converting it to lower case.

BTW, when you are adding or editing translatable strings, you do it only in the English version. See the _README* files in the other language directories for more information.